### DISCUSSION TITLE
Merge both sidebars. (Firefox already released their own sidebar, why needs another?)

### DISCUSSION THREAD

**Discussion by shawny43** [zen-browser/desktop#3530] [Ideas]
My question is why needs two sidebars when you already have one. If any zen devs could merge codes together and make one sidebar instead of both zen side and Firefox side could maybe help. Also Firefox Labs include some AI features that require using the native sidebar and others extensions has well. I guess you may needs to merge both and make sure all extensions and cores functionalities using Zen Sidebar instead of the native one. 

![image](https://github.com/user-attachments/assets/9d7d7972-7621-41f1-a930-8d5273f1f3e5)
- Zen Sidebar in left.
- Native Firefox in right. 
- Center is the context page. 

Also both sidebars icons are identical wich could confuse when you have to edit your toolbar to make you feel confortable. Should create an specific icon for zen sidebar? Maybe also rename Sidebar for zen side to Sidekick, SideDash, Magic Bar or again maybe Zen Bar?

There a long time ago no one notice there two with the same name and icons in Zen Browsers wich share the same functions except the zen side with tiny tabs and the other one using extensions and core functions.  

There some ideas concept. Ignore the both but imagine just make one side a zen and another side a fox. Made it myself has concept. 
![zenfoxbar](https://github.com/user-attachments/assets/97de095b-0e97-4a47-aca1-4bbb8db44440)

The objective of the idea concept is to help peoples to understood what's sidebar they are clicking and which one is native and where is Zen side. This may could help Zen Devs.

**Comment by shawny43** [comment]
Sorry to say that, but i have to bump since Zen Team didn't manage to answer their opinion to this post. Also having both sidebar are just not very intuitive and may break the use of Zen sidebar if Firefox Sidebar are here too.

**Comment by mia-riezebos** [comment]
I don't agree with this idea. If this is planned or implemented it should be optional, or there should be a way to open multiple sidebars. Currently my sidebar is permanently occupied with Sidebery, as a tab bar replacement, but I still like to use web panels as an ephemeral sidebar to quickly check on my music, or chats while browsing.

**Reply by shawny43** [reply]
When  you open settings is already occupied a tab. Has i saw on FF side it's seem you can switch between window on sidebar like switch chatbot to sideberry and vice-versa like tabs. So i don't think it's should be problematic. Merging not mean we have to remove firefox core himselft. We can merge by sharing the same sidebar with ease without issue. The issue is how we could replicate the zen formula from tabs splitting to sidebars like split-screens somehow, also the bottoms icons could be a nice place to replicate some like Vivaldi and Opera did wich extensions could be act like sidebar tabs instead of just a dropdown buttons next to sidebar webpages. If sideberry is really important for you to replace Zen side, you should ask Zen to add sideberry native support like Waterfox or Floorp did. That all i could say about it.

**Reply by mia-riezebos** [reply]
I specifically like being able to leave Sidebery open and being able to then open up a second sidebar for music controls. merging them would result in a sidebar that has to constantly switch between my tab manager, and other sidebar stuff. When Sidebery is toggled off, the Sidebery shortcuts do not work (so I can't open or close tabs without sidebery open)

**Reply by mia-riezebos** [reply]
I agree that the terminology and everything is very confusing and should be improved. 
My suggestion would be to have a dedicated "Left Sidebar" and "Right Sidebar"

and then you can put whatever you want in either sidebar

**Reply by mia-riezebos** [reply]
I think this could be a good solution to the lack of comprehensibility and get rid of redundant terminology

**Reply by shawny43** [reply]
> I think this could be a good solution to the lack of comprehensibility and get rid of redundant terminology

True, i guess calling panels instead of sidebar could be a nice improvement to prevent misleading stuffs. Also Vivaldi calls side webs has panels anyways so i don't think it's could be a issue.

**Reply by John-Gee** [reply]
Thanks for your comment @mia-riezebos ,  I didn't know about Sidebery or alike before, it looks like I was stuck in the early 2000s with my browsing now.

**Comment by menganatha** [comment]
Firefox had it's own sidebar since it's release and a 2nd one was always needed since naturally the already existing one was occupied by sidebery or a similar addon.
>Firefox already released their own sidebar

As far as I understand this new sidebar from labs menu is just a tiny bit less neutered version of the original sidebar, that's it. No additional sidebar was added.
As to prevent confusion, you can call zen sidebar a sidepanel, like many do.
